### ISSUE TITLE
LOSS 显示为NAN

### ISSUE THREAD

**Issue by charlesfufu** [GaoQ1/information-demo#2]
楼主有遇到过训练的时候loss突然为Nan这种情况过吗

**Comment by GaoQ1** [comment]
你交叉熵损失函数用的什么，可以考虑换个

**Comment by charlesfufu** [comment]
> 你交叉熵损失函数用的什么，可以考虑换个

用了softmax_cross_entropy

**Comment by GaoQ1** [comment]
那你换sparse_softmax_cross_entropy_with_logits这个试试，softmax_cross_entropy是经常出现NaN

**Comment by charlesfufu** [comment]
> 那你换sparse_softmax_cross_entropy_with_logits这个试试，softmax_cross_entropy是经常出现NaN

楼主知道为什么会出现这种情况呢？两个交叉熵函数按理没有本质区别的。

**Comment by charlesfufu** [comment]
> 那你换sparse_softmax_cross_entropy_with_logits这个试试，softmax_cross_entropy是经常出现NaN

我最后用了梯度裁剪貌似可以了，loss输出没有Nan了

**Comment by GaoQ1** [comment]
https://stackoverflow.com/questions/45889951/tensorflow-loss-value-is-nan-with-sparse-softmax-cross-entropy-with-logits 这个上面有说loss nan的可能情况。这两个交叉熵函数区别在于label的稀疏与否，没多大差别，我上面说的有问题，只是印象中有这么回事。
